AndroidAnnotations/androidannotations-rest-spring/rest-spring/src/main/java/org/androidannotations/rest/spring/helper/RestSpringValidatorHelper.java

```diff
@@ -423,4 +517,42 @@ public void hasSetOfHttpMethodReturnType(ExecutableElement element, ElementValid
 			}
 		}
 	}
+
+	public void doesNotHavePathParamAnnotation(Element element, ElementValidation validation) {
```

Comment:
The method should be named `doesNotHavePathAnnotation` because the annotation is `@Path`.